tests/test__data.py

```diff
@@ -220,6 +220,20 @@ def setUp(self):
         r = f.reduced_chi_squareds()
         self.assertIs(type(r), list)
         self.assertAlmostEqual(r[0], 1.5, 1)
+
+        # Try supplying a callable function
+        def tf(x, a1, a2, a3):
```

Comment:
Happy to switch to more descriptive names if that's preferred. I was following the convention used elsewhere in the test suite (specifically, https://github.com/Spinmob/spinmob/blob/master/tests/test__data.py#L210-L217) of very short names everywhere, just for consistency.